.github/workflows/ci.yml

```diff
@@ -138,6 +138,10 @@ jobs:
         if: matrix.scala != '3.1.2'
         run: sbt 'project ${{ matrix.project }}' '++${{ matrix.scala }}' '${{ matrix.ci }}' 'scalafixAll --check'
 
+      - name: Check scalafix lints
+        if: matrix.java == 'temurin@8' && !startsWith(matrix.scala, '3.')
+        run: sbt 'project ${{ matrix.project }}' '++${{ matrix.scala }}' 'scalafixAll --check'
+
```

Comment:
We should remove the one defined in our build, now that we get it from the plugin.